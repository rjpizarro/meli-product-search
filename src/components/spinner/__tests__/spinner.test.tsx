import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../index'

describe('Spinner', () => {
    it('should render correctly ', () => {
        const tree = renderer
            .create(<Spinner />)
            .toJSON()

        expect(tree).toMatchSnapshot()
    })
})

