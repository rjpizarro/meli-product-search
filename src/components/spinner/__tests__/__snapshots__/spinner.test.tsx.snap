// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly  1`] = `
<div
  className="spinner"
/>
`;
